compiler/rustc_codegen_llvm/src/llvm/ffi.rs

```diff
@@ -1868,7 +1868,10 @@ extern "C" {
     pub fn LLVMRustGetLastError() -> *const c_char;
 
     /// Print the pass timings since static dtors aren't picking them up.
-    pub fn LLVMRustPrintPassTimings();
+    pub fn LLVMRustPrintPassTimings(size: *const usize) -> *const c_char;
```

Comment:
Nothing else in this file uses `usize`, please use `size_t` to make sure we don't rely on funny Rust<->C differences